dice-game-ui-angular/src/app/services/local-game-storage-service.ts

```diff
@@ -42,14 +53,22 @@ export class LocalGameStorageService implements GameStorageService {
     if (!curSavedGame) {
       throw new Error(`A game with the guid ${guid.toString()} wasn't found in the stored games`);
     }
-    return new SimpleGameEngine(curSavedGame.gameEngine.fieldSize.width, curSavedGame.gameEngine.fieldSize.height);
+    return new SimpleGameEngine(
+      curSavedGame.gameEngine.fieldSize.width,
+      curSavedGame.gameEngine.fieldSize.height,
+      this.user.email
+    );
   }
 
-  getAllSavedGames(): SavedGameEngine[] {
+  protected getAllSavedGames(): SavedGameEngine[] {
```

Comment:
Protected methods should be located after all public methods.

Reply:
Fixed